app/templates/gentelella/guest/event/cfs.html

```diff
@@ -62,6 +62,15 @@ <h4 class="modal-title">{{ _("Please login to submit your proposal") }} </h4>
                                         <h4 class="modal-title">{{ _("Submit Proposal") }}? </h4>
                                     </div>
                                     <div class="modal-footer" style="padding:6px">
+                                        <div class="existing-session" style="text-align:left">
+                                            <h4>{{ _("Existing Sessions") }} </h4>
```

Comment:
I think this should have a subcontent of "no submitted session" message if no session. Cause it would make it a little clear?